doc/lua_api.md

```diff
@@ -7740,6 +7740,18 @@ child will follow movement and rotation of that bone.
                 abides by, `"custom"` uses `sun_tint` and `moon_tint`, while
                 `"default"` uses the classic Minetest sun and moon tinting.
                 Will use tonemaps, if set to `"default"`. (default: `"default"`)
+        * `fog`: A table with following optional fields:
+            * `fog_distance`: integer, set an upper bound the client's viewing_range (inluding range_all).
+               By default, fog_distance is controlled by the client's viewing_range, and this field is not set.
+               Any value >= 0 sets the desired upper bound for the client's viewing_range and disables range_all.
+               Any value < 0, resets the behavior to being client-controlled.
+               (default: not set)
```

Comment:
Outdated doc.
```suggestion
               (default: `-1`)
```
(Same below.)

Reply:
The default remains... Not set. I.e. unless you actually provide a value (like -1) the existing settings remains.